src/containers/pages/IRSLite/MapLite/index.tsx

```diff
@@ -375,6 +375,7 @@ const IRSLiteReportingMap = (
                   {!row.listDisplay && <p className="indicator-description">{row.description}</p>}
                   {!row.listDisplay && (
                     <ProgressBar
+                      lineColor={indicatorThresholdsIRSLite.GREEN_THRESHOLD.color}
```

Comment:
IIRC this will set the default line color to Green - right?  I'm not sure that is what we want in this case.  Instead, we should leave the default "as is" and instead ensure that when the value is > 100%, the color rendered is green.

Reply:
@moshthepitt I have tested this implementation, 
Issue: 
- Before reaching 100, it shows gray color, and when >100 it shows blue color which was set to default.

So changing default color worked.

Reply:
Thanks @haseemVD - is it possible to capture that in a test? i.e. that it is grey before 100 and then green when > 100?

Reply:
@moshthepitt Added test,

Just a thought:
- We are using `ProgressBar` from `@onaio/progress-indicators`, So these color changing tests should be written in that package to check if thats working fine or not when passing `lineColor` as a porp.

Reply:
From the Real point of view, we care about the color being grey or green or whatever else.  So that's how I would justify writing these tests on this side.  The tests on the `@onaio/progress-indicators` side should be to ensure that that component works as intended in general.